src/dotnet/UI.Blazor/Components/AccountDropdown/SignedInMenu.razor

```diff
@@ -1,24 +1,38 @@
 ﻿@namespace ActualChat.UI.Blazor.Components
 @using ActualChat.Users
 @using ActualChat.UI.Blazor.Services
+@using ActualChat.Hosting
+
+@{
+    var version = "v" + (GetType().Assembly.GetInformationalVersion() ?? "n/a").Replace('+', ' ');
```

Comment:
It's better to cache it in `private static readonly Version { get; }` & initialize it from type initializer or in-place. Doing this on every render is a waste.